### PR TITLE
Bump/oa v0.6.0

### PR DESCRIPTION
**Pull request by 0xArdi**
This PR bumps the `autonomous-fund` to `open-autonomy@v0.6.0`.

### CODE REVIEW

File: packages/balancer/agents/autonomous_fund/aea-config.yaml [hunk 1]
```diff
@@ -69,6 +69,7 @@ logging_config:
       - logfile
       - console
       propagate: true
+skill_exception_policy: stop_and_exit
```
Comment (0xArdi) [inline packages/balancer/agents/autonomous_fund/aea-config.yaml:72]: otherwise exception logs dont show on the logfiles

**Review by Adamantios — APPROVED** [review]

File: packages/balancer/skills/pool_manager_abci/payloads.py [hunk 2]
```diff
@@ -37,38 +36,43 @@ def __str__(self) -> str:
...
+
+    @property
+    def update_pool_tx(self) -> str:
+        """Get the UpdatePool transaction data."""
+        return self._update_pool_tx
+
+    @property
+    def data(self) -> Dict:
+        """Get the data."""
+        return dict(update_pool_tx=self.update_pool_tx)
```
[the remark this answers is not in the document]
Comment (DavidMinarsch) [reply packages/balancer/skills/pool_manager_abci/payloads.py:78]: Yes the "new" version should be removed from the scaffolding. Suggest you open an issue.

File: packages/balancer/skills/pool_manager_abci/rounds.py [hunk 3]
```diff
@@ -186,3 +181,15 @@ class PoolManagerAbciApp(AbciApp[Event]):
...
     cross_period_persisted_keys: List[str] = []
+    db_pre_conditions: Dict[AppState, List[str]] = {
+        DecisionMakingRound: [
+            get_name(SynchronizedData.most_voted_estimates),
+        ],
+    }
+    db_post_conditions: Dict[AppState, List[str]] = {
+        FinishedTxPreparationRound: [
+            get_name(SynchronizedData.most_voted_tx_hash),
+            get_name(SynchronizedData.safe_contract_address),
```
Comment (DavidMinarsch) [inline packages/balancer/skills/pool_manager_abci/rounds.py:192]: The `safe_contract_address` is not "produced" in this skill, so shouldn't be a post-condition.

Comment (0xArdi) [reply packages/balancer/skills/pool_manager_abci/rounds.py:192]: There's an error if I don't add it here. It seems the `FinishedRegistrationRound` branch is not being traversed. 
```
Error: Pre conditions '{'safe_contract_address'}' of app '<class 'packages.valory.skills.transaction_settlement_abci.rounds.TransactionSubmissionAbciApp'>' not a post condition of app '<class 'packages.balancer.skills.pool_manager_abci.rounds.PoolManagerAbciApp'>' or any preceding app in path [(<class 'packages.valory.skills.registration_abci.rounds.RegistrationStartupRound'>, <class 'packages.valory.skills.registration_abci.rounds.AgentRegistrationAbciApp'>, <class 'packages.valory.skills.registration_abci.rounds.FinishedRegistrationFFWRound'>), (<class 'packages.balancer.skills.fear_and_greed_oracle_abci.rounds.ObservationRound'>, <class 'packages.balancer.skills.fear_and_greed_oracle_abci.rounds.FearAndGreedOracleAbciApp'>, <class 'packages.balancer.skills.fear_and_greed_oracle_abci.rounds.FinishedDataCollectionRound'>), (<class 'packages.balancer.skills.pool_manager_abci.rounds.DecisionMakingRound'>, <class 'packages.balancer.skills.pool_manager_abci.rounds.PoolManagerAbciApp'>, <class 'packages.balancer.skills.pool_manager_abci.rounds.FinishedTxPreparationRound'>), (<class 'packages.valory.skills.transaction_settlement_abci.rounds.RandomnessTransactionSubmissionRound'>, <class 'packages.valory.skills.transaction_settlement_abci.rounds.TransactionSubmissionAbciApp'>, <class 'packages.valory.skills.transaction_settlement_abci.rounds.FinishedTransactionSubmissionRound'>), (<class 'packages.valory.skills.reset_pause_abci.rounds.ResetAndPauseRound'>, <class 'packages.valory.skills.reset_pause_abci.rounds.ResetPauseAbciApp'>, None)].

```

Comment (DavidMinarsch) [reply packages/balancer/skills/pool_manager_abci/rounds.py:192]: Then open an issue please

Comment (0xArdi) [reply packages/balancer/skills/pool_manager_abci/rounds.py:192]: https://github.com/valory-xyz/open-autonomy/issues/1681

File: packages/balancer/agents/autonomous_fund/aea-config.yaml (hunk 1 above)
Comment (0xArdi) [reply packages/balancer/agents/autonomous_fund/aea-config.yaml:72]: https://github.com/valory-xyz/open-autonomy/issues/1680

File: packages/balancer/skills/pool_manager_abci/payloads.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (0xArdi) [reply packages/balancer/skills/pool_manager_abci/payloads.py:78]: https://github.com/valory-xyz/open-autonomy/issues/1680

File: packages/balancer/skills/pool_manager_abci/rounds.py (hunk 3 above)
Comment (DavidMinarsch) [reply packages/balancer/skills/pool_manager_abci/rounds.py:192]: Looks to me like the same issue as here: https://github.com/valory-xyz/price-oracle/pull/24/files - although still need to confirm.

Comment (DavidMinarsch) [reply packages/balancer/skills/pool_manager_abci/rounds.py:192]: Looks the same:

```
Error: Pre conditions '{'safe_contract_address'}' of app '<class 'packages.valory.skills.transaction_settlement_abci.rounds.TransactionSubmissionAbciApp'>' not a post condition of app '<class 'packages.balancer.skills.pool_manager_abci.rounds.PoolManagerAbciApp'>' or any preceding app in path

[
(<class 'packages.valory.skills.registration_abci.rounds.RegistrationStartupRound'>, <class 'packages.valory.skills.registration_abci.rounds.AgentRegistrationAbciApp'>, <class 'packages.valory.skills.registration_abci.rounds.FinishedRegistrationFFWRound'>),

(<class 'packages.balancer.skills.fear_and_greed_oracle_abci.rounds.ObservationRound'>, <class 'packages.balancer.skills.fear_and_greed_oracle_abci.rounds.FearAndGreedOracleAbciApp'>, <class 'packages.balancer.skills.fear_and_greed_oracle_abci.rounds.FinishedDataCollectionRound'>),

(<class 'packages.balancer.skills.pool_manager_abci.rounds.DecisionMakingRound'>, <class 'packages.balancer.skills.pool_manager_abci.rounds.PoolManagerAbciApp'>, <class 'packages.balancer.skills.pool_manager_abci.rounds.FinishedTxPreparationRound'>),

(<class 'packages.valory.skills.transaction_settlement_abci.rounds.RandomnessTransactionSubmissionRound'>, <class 'packages.valory.skills.transaction_settlement_abci.rounds.TransactionSubmissionAbciApp'>, <class 'packages.valory.skills.transaction_settlement_abci.rounds.FinishedTransactionSubmissionRound'>),

(<class 'packages.valory.skills.reset_pause_abci.rounds.ResetAndPauseRound'>, <class 'packages.valory.skills.reset_pause_abci.rounds.ResetPauseAbciApp'>, None)
].
```
this path is not possible without specifying the appropriate post condition of the registration skill.

**Pull request merged**